### PR TITLE
Mainly build process & artifacts

### PR DESCRIPTION
**Pull request by meeDamian**
This PR contains:

* Update Go to 1.13
* Deterministic builds 🎉
* `arm64` support
* Switch to Docker's new BUILDKIT build engine
* Remove BSD binaries
* Swich CI from Travis to [Github Actions]

### Note on determinism

This PR uses Docker stages to concurrently build `invoicer` for each supported platform, on two different operating systems: Alpine and Debian, and then uses a 3rd stage to fetch both and compare if output binaries are strictly identical.  And only if they are the `final` stage is created.

[Github Actions]: https://github.com/lncm/invoicer/tree/0d06e4d1d17203365ca76c3842bebbd1616f72bf/.github/workflows

---
This change is [Reviewable](https://reviewable.io/reviews/lncm/invoicer/43)

### CODE REVIEW

**Comment by meeDamian** [comment]
Since Travis doesn't seems to like Buildkit for whatever reason… I reckon the switch to Github Actions might get a little accelerated…

**Comment by meeDamian** [comment]
Alright, finished and awaiting review :). cc. @AnotherDroog @nolim1t

**Comment by meeDamian** [comment]
@noaho You might want to take a look too, as this PR is primarily a CI/Docker change.

Basically Travis gets replaced with Github Actions, and Dockerfile is expanded to build the same on different OSs to verify the output is identical later.

**Comment by nolim1t** [comment]
lgtm

File: scripts/ci-set-target-image-architecture.sh [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  exit 0
+fi
+
+# If `gsed` is available in the system, then use it instead as the available `sed` might not be too able (MacOS)…
+SED="sed"
+if command -v gsed >/dev/null; then
+  SED=gsed
+fi
+
+# Decyphering `sed` expressions is always "fun"…  So to make it easier on the reader here's an explanation:
```
Comment (AnotherDroog) [inline scripts/ci-set-target-image-architecture.sh]: I really like this comment to make some of the magic more comprehensible!

**Comment by meeDamian** [comment]
Released as `v0.6.0`.

Built log here: https://github.com/lncm/invoicer/runs/224216015
Release: https://github.com/lncm/invoicer/releases/tag/v0.6.0
Docker Hub: https://hub.docker.com/r/lncm/invoicer/tags?page=1&ordering=last_updated

**Pull request merged**